modules/wordads/wordads.php

```diff
@@ -92,6 +92,23 @@ function init() {
 		}
 
 		$this->insert_adcode();
+
+		if ( '/ads.txt' === esc_url_raw( $_SERVER['REQUEST_URI'] ) ) {
+			header( 'Content-Type: text/plain; charset=utf-8' );
+
+			/**
+			 * Provide plugins a way of modifying the contents of the automatically-generated ads.txt file.
+			 *
+			 * @module wordads
+			 *
+			 * @since 6.1.0
+			 *
+			 * @param string WordAds_API::get_wordads_ads_txt() The contents of the ads.txt file.
+			 */
+			$ads_txt_content = apply_filters( 'wordads_ads_txt', WordAds_API::get_wordads_ads_txt() );
```

Comment:
This is a great note.

I've updated the code to pass a blank string to the filter in case of `WP_Error`.
This would allow the additional contents added via filter to still be output in the case of a WordPress.com API failure.